subsystem/kubeapi_test.go

```diff
@@ -1643,6 +1643,9 @@ var _ = Describe("[kube-api]cluster installation", func() {
 			Expect(err).To(BeNil())
 			return h.InstallationDiskID
 		}, "2m", "10s").Should(Equal(sdb.ID))
+		Eventually(func() string {
+			return getAgentCRD(ctx, kubeClient, key).Status.InstallationDiskID
```

Comment:
Since we already expect in the agent_controller_test that `agent.Spec.InstallationDiskID` is equal to the installation disk ID, is it also reasonable to add a check here?

Reply:
Very good idea, modified so we now check
1) value of `InstallationDiskID` in the DB
2) value of above in Spec
3) value of above in Status